config/locales/en-DENVER.yml

```diff
@@ -58,8 +93,6 @@ en-DENVER:
         label: Role
       contributor_grid:
         label: GRID
-      contributor_role:
-        label: Role
```

Comment:
Any reason for removing these lines??

Reply:
Ok, thats super wierd. My editor flagged them as duplicate, so i removed the extra one, but now its saying that there is only one. 🤔

Reply:
This is what i'm seeing. 
![image](https://user-images.githubusercontent.com/1199977/142895543-e86852b2-a3e6-48ca-9b90-4d0283e01877.png)